onnxruntime/contrib_ops/cuda/fused_conv.cc

```diff
@@ -38,35 +38,37 @@ class FusedConv : public onnxruntime::cuda::Conv<T, false> {
     CUDNN_RETURN_IF_ERROR(status_);
     std::lock_guard<OrtMutex> lock(Base::s_.mutex);
     auto cudnnHandle = this->GetCudnnHandle(context);
-    ORT_RETURN_IF_ERROR(Base::UpdateState(context, true));
+    ORT_RETURN_IF_ERROR(Base::UpdateState(context));
     if (Base::s_.Y->Shape().Size() == 0) {
       return Status::OK();
     }
     bool has_z = nullptr != Base::s_.z_data;
     bool has_b = nullptr != Base::s_.b_data;
     typedef typename onnxruntime::cuda::ToCudaType<T>::MappedType CudaT;
-    const auto alpha = onnxruntime::cuda::Consts<CudaT>::One;
-    const auto beta = onnxruntime::cuda::Consts<CudaT>::Zero;
+    static const auto alpha = onnxruntime::cuda::Consts<CudaT>::One;
+    static const auto beta = onnxruntime::cuda::Consts<CudaT>::Zero;
     IAllocatorUniquePtr<void> workspace = Base::GetWorkSpace(context->GetComputeStream());
-    auto cudnn_status = cudnnConvolutionBiasActivationForward(cudnnHandle,
-                                                              &alpha,
-                                                              Base::s_.x_tensor,
-                                                              Base::s_.x_data,
-                                                              Base::s_.w_desc,
-                                                              Base::s_.w_data,
-                                                              Base::s_.conv_desc,
-                                                              Base::s_.algo,
-                                                              workspace.get(),
-                                                              Base::s_.workspace_bytes,
-                                                              has_z ? &alpha : &beta,
-                                                              has_z ? Base::s_.z_tensor : Base::s_.y_tensor,
-                                                              has_z ? Base::s_.z_data : Base::s_.y_data,
-                                                              Base::s_.b_tensor,
-                                                              has_b ? Base::s_.b_data : Base::s_.b_zero,
-                                                              activation_desc_,
-                                                              Base::s_.y_tensor,
-                                                              Base::s_.y_data);
-    if (CUDNN_STATUS_SUCCESS != cudnn_status) {
+
+    if (has_b && !Base::s_.post_slicing_required) {
```

Comment:
The condition here was:

"if (has_b && has_z && !Base::s_.post_slicing_required)"

the "has_z" blocked inception models from being optimized to the best, which caused perf regression.